src/modules/esl-traversing-query/core/esl-traversing-query.ts

```diff
@@ -132,18 +132,21 @@ export class TraversingQuery {
     return this.traverseChain(initial, tuple(parts), findFirst);
   }
 
-  /** @returns first matching element reached via {@link TraversingQuery} rules */
+  /** @returns first matching element reached via {@link ESLTraversingQuery} rules */
   static first(query: string, base?: Element | null, scope?: Element): Element | null {
-    return TraversingQuery.traverse(query, true, base, scope)[0] || null;
+    return ESLTraversingQuery.traverse(query, true, base, scope)[0] || null;
   }
-  /** @returns Array of all matching elements reached via {@link TraversingQuery} rules */
+  /** @returns Array of all matching elements reached via {@link ESLTraversingQuery} rules */
   static all(query: string, base?: Element | null, scope?: Element): Element[] {
-    return TraversingQuery.traverse(query, false, base, scope);
+    return ESLTraversingQuery.traverse(query, false, base, scope);
   }
 }
 
+/** @deprecated alias for {@link ESLTraversingQuery} */
+export const TraversingQuery = ESLTraversingQuery;
+
 declare global {
   export interface ESLLibrary {
-    TraversingQuery: typeof TraversingQuery;
+    TraversingQuery: typeof ESLTraversingQuery;
```

Comment:
In global scope we still use ESL.TraversingQuery, don't we?

Reply:
Nope, `@ExportNs` stores everything differently (with the name defined in decorator params) and always under ESL "namespace", so globally it will be `ESL.Traversing`

Reply:
That's exactly what I was asking. Thanks!